cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -133,7 +133,7 @@ function doFormat(opts) {
     const {prefix, equalString, epsilon, allowFuture, short} = opts,
         diff = opts.relativeTo - opts.timestamp,
         elapsed = Math.abs(diff),
-        isEqual =  elapsed < epsilon * SECONDS,
+        isEqual = elapsed <= (epsilon ?? 0) * SECONDS,
```

Comment:
Why the `?? 0` here? We default it to 10 above, do we expect someone to pass `null`?